common-custom-user-data-maven-extension/src/main/java/com/gradle/maven/common/user/data/extension/CustomBuildScanEnhancements.java

```diff
@@ -1,35 +1,37 @@
-package com.gradle;
+package com.gradle.maven.common.user.data.extension;
```

Comment:
Maybe a bit shorter?
```suggestion
package com.gradle.maven.custom.extension;
```

Reply:
I went back with the existing package  `com.gradle`. We originally needed to change it, because I was importing it back to the GE extension realm, and having sth not precise enough was causing some recursion in class loading. This is not a problem anymore as we don't import anything back